[gdal] Build failure

Package: gdal:x64-windows -> 3.7.1#3

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 4a600e9fe 2023-09-01 (4 days ago)

**To Reproduce**

`vcpkg x-set-installed gdal[core]`
**Failure logs**

```
-- Using cached OSGeo-gdal-v3.7.1.tar.gz.
-- Extracting source C:/v/downloads/OSGeo-gdal-v3.7.1.tar.gz
-- Applying patch find-link-libraries.patch
-- Applying patch fix-gdal-target-interfaces.patch
-- Applying patch libkml.patch
-- Applying patch Fix-quote-variable.patch
-- Using source at C:/v/vcpkg1/buildtrees/gdal/src/v3.7.1-e612b9c528.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j31
    Working Directory: C:/v/vcpkg1/buildtrees/gdal/x64-windows-dbg
    See logs for more information:
      C:\v\vcpkg1\buildtrees\gdal\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/gdal/portfile.cmake:104 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\v\vcpkg1\buildtrees\gdal\install-x64-windows-dbg-out.log</summary>

```
Change Dir: 'C:/v/vcpkg1/buildtrees/gdal/x64-windows-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j31 install
[1/896] cmd.exe /C "cd /D C:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -DSOURCE_DIR=C:/v/vcpkg1/buildtrees/gdal/src/v3.7.1-e612b9c528.clean -DBINARY_DIR=C:/v/vcpkg1/buildtrees/gdal/x64-windows-dbg -DGDAL_SHA1SUM= -DGDAL_RELEASE_DATE= -P C:/v/vcpkg1/buildtrees/gdal/src/v3.7.1-e612b9c528.clean/cmake/helpers/generate_gdal_version_h.cmake"
[2/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /wd4244 /wd4702 /wd4701 /wd4013 /wd4706 /wd4057 /wd4210 /wd4305 /wd4131 /showIncludes /Fofrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\inftree9.c.obj /Fdfrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\zlib\contrib\infback9\inftree9.c
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[3/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /wd4244 /wd4702 /wd4701 /wd4013 /wd4706 /wd4057 /wd4210 /wd4305 /wd4131 /showIncludes /Fofrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\minified_zutil.c.obj /Fdfrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\zlib\contrib\infback9\minified_zutil.c
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[4/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /wd4244 /wd4702 /wd4701 /wd4013 /wd4706 /wd4057 /wd4210 /wd4305 /wd4131 /showIncludes /Fofrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\infback9.c.obj /Fdfrmts\zlib\contrib\infback9\CMakeFiles\infback9.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\zlib\contrib\infback9\infback9.c
...
Skipped 3696 lines
...
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2628): warning C4251: 'GDALAttributeString::m_osValue': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALAttributeString'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2654): warning C4251: 'GDALAttributeNumeric::m_dims': class 'std::vector<std::shared_ptr<GDALDimension>,std::allocator<std::shared_ptr<GDALDimension>>>' needs to have dll-interface to be used by clients of class 'GDALAttributeNumeric'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2325): note: see declaration of 'std::vector<std::shared_ptr<GDALDimension>,std::allocator<std::shared_ptr<GDALDimension>>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2658): warning C4251: 'GDALAttributeNumeric::m_anValuesUInt32': class 'std::vector<GUInt32,std::allocator<GUInt32>>' needs to have dll-interface to be used by clients of class 'GDALAttributeNumeric'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2658): note: see declaration of 'std::vector<GUInt32,std::allocator<GUInt32>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2721): warning C4251: 'GDALMDArray::m_poCachedArray': class 'std::shared_ptr<GDALMDArray>' needs to have dll-interface to be used by clients of class 'GDALMDArray'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(1496): note: see declaration of 'std::shared_ptr<GDALMDArray>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2963): warning C4251: 'GDALMDArrayRegularlySpaced::m_dims': class 'std::vector<std::shared_ptr<GDALDimension>,std::allocator<std::shared_ptr<GDALDimension>>>' needs to have dll-interface to be used by clients of class 'GDALMDArrayRegularlySpaced'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2325): note: see declaration of 'std::vector<std::shared_ptr<GDALDimension>,std::allocator<std::shared_ptr<GDALDimension>>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2964): warning C4251: 'GDALMDArrayRegularlySpaced::m_attributes': class 'std::vector<std::shared_ptr<GDALAttribute>,std::allocator<std::shared_ptr<GDALAttribute>>>' needs to have dll-interface to be used by clients of class 'GDALMDArrayRegularlySpaced'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2245): note: see declaration of 'std::vector<std::shared_ptr<GDALAttribute>,std::allocator<std::shared_ptr<GDALAttribute>>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(2965): warning C4251: 'GDALMDArrayRegularlySpaced::m_osEmptyFilename': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALMDArrayRegularlySpaced'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3092): warning C4251: 'GDALDimension::m_osName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALDimension'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3093): warning C4251: 'GDALDimension::m_osFullName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALDimension'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3094): warning C4251: 'GDALDimension::m_osType': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALDimension'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3095): warning C4251: 'GDALDimension::m_osDirection': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALDimension'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3107): warning C4251: 'GDALDimensionWeakIndexingVar::m_poIndexingVariable': class 'std::weak_ptr<GDALMDArray>' needs to have dll-interface to be used by clients of class 'GDALDimensionWeakIndexingVar'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3107): note: see declaration of 'std::weak_ptr<GDALMDArray>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3170): warning C4251: 'GDALRelationship::m_osName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3171): warning C4251: 'GDALRelationship::m_osLeftTableName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3172): warning C4251: 'GDALRelationship::m_osRightTableName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3175): warning C4251: 'GDALRelationship::m_osMappingTableName': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3176): warning C4251: 'GDALRelationship::m_osListLeftTableFields': class 'std::vector<std::string,std::allocator<std::string>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogr_feature.h(732): note: see declaration of 'std::vector<std::string,std::allocator<std::string>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3177): warning C4251: 'GDALRelationship::m_osListRightTableFields': class 'std::vector<std::string,std::allocator<std::string>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogr_feature.h(732): note: see declaration of 'std::vector<std::string,std::allocator<std::string>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3178): warning C4251: 'GDALRelationship::m_osListLeftMappingTableFields': class 'std::vector<std::string,std::allocator<std::string>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogr_feature.h(732): note: see declaration of 'std::vector<std::string,std::allocator<std::string>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3179): warning C4251: 'GDALRelationship::m_osListRightMappingTableFields': class 'std::vector<std::string,std::allocator<std::string>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogr_feature.h(732): note: see declaration of 'std::vector<std::string,std::allocator<std::string>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3181): warning C4251: 'GDALRelationship::m_osForwardPathLabel': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3182): warning C4251: 'GDALRelationship::m_osBackwardPathLabel': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdal_priv.h(3183): warning C4251: 'GDALRelationship::m_osRelatedTableType': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'GDALRelationship'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\xstring(5119): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\mdreader\reader_pleiades.h(66): warning C4251: 'GDALMDReaderPleiades::m_osBaseFilename': class 'CPLString' needs to have dll-interface to be used by clients of class 'GDALMDReaderPleiades'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port\cpl_string.h(311): note: see declaration of 'CPLString'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\mdreader\reader_pleiades.h(67): warning C4251: 'GDALMDReaderPleiades::m_osIMDSourceFilename': class 'CPLString' needs to have dll-interface to be used by clients of class 'GDALMDReaderPleiades'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port\cpl_string.h(311): note: see declaration of 'CPLString'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\mdreader\reader_pleiades.h(68): warning C4251: 'GDALMDReaderPleiades::m_osRPBSourceFilename': class 'CPLString' needs to have dll-interface to be used by clients of class 'GDALMDReaderPleiades'
C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port\cpl_string.h(311): note: see declaration of 'CPLString'
[881/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdal_grid_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdal_grid_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[882/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -DPAM_ENABLED -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\gtiff -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\mem -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Fogcore\CMakeFiles\gcore.dir\overview.cpp.obj /Fdgcore\CMakeFiles\gcore.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\overview.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[883/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -DPAM_ENABLED -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\gtiff -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\mem -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Fogcore\CMakeFiles\gcore.dir\rasterio.cpp.obj /Fdgcore\CMakeFiles\gcore.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\rasterio.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[884/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdaldem_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdaldem_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[885/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdal_rasterize_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdal_rasterize_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[886/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdalinfo_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdalinfo_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[887/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\ogrinfo_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\ogrinfo_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[888/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdalmdiminfo_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdalmdiminfo_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[889/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdal_translate_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdal_translate_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[890/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdalbuildvrt_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdalbuildvrt_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[891/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\ogr2ogr_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\ogr2ogr_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[892/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdalwarp_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdalwarp_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[893/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -DPAM_ENABLED -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\gtiff -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\mem -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Fogcore\CMakeFiles\gcore.dir\gdalmultidim.cpp.obj /Fdgcore\CMakeFiles\gcore.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore\gdalmultidim.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[894/896] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\alg -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\gcore -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\gcore -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\port -IC:\v\vcpkg1\buildtrees\gdal\x64-windows-dbg\port -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\frmts\vrt -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\generic -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\geojson -IC:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\ogr\ogrsf_frmts\mem -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include -IC:\v\vcpkg1\vcpkg_installed\x64-windows\include\json-c /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /EHsc /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 /showIncludes /Foapps\CMakeFiles\appslib.dir\gdalmdimtranslate_lib.cpp.obj /Fdapps\CMakeFiles\appslib.dir\ /FS -c C:\v\vcpkg1\buildtrees\gdal\src\v3.7.1-e612b9c528.clean\apps\gdalmdimtranslate_lib.cpp
cl : Command line warning D9025 : overriding '/W3' with '/W4'
[895/896] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=CMakeFiles\GDAL.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\link.exe  @CMakeFiles\GDAL.rsp  /out:gdald.dll /implib:gdald.lib /pdb:gdald.pdb /dll /version:37.0 /machine:x64 /nologo    /debug /INCREMENTAL  && cd ."
FAILED: gdald.dll gdald.lib 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=CMakeFiles\GDAL.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\link.exe  @CMakeFiles\GDAL.rsp  /out:gdald.dll /implib:gdald.lib /pdb:gdald.pdb /dll /version:37.0 /machine:x64 /nologo    /debug /INCREMENTAL  && cd ."
LINK Pass 1: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\link.exe @CMakeFiles\GDAL.rsp /out:gdald.dll /implib:gdald.lib /pdb:gdald.pdb /dll /version:37.0 /machine:x64 /nologo /debug /INCREMENTAL /MANIFEST /MANIFESTFILE:CMakeFiles\GDAL.dir/intermediate.manifest CMakeFiles\GDAL.dir/manifest.res" failed (exit code 1120) with the following output:
   Creating library gdald.lib and object gdald.exp
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_std_error referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_CreateCompress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_CreateDecompress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_destroy_compress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_destroy_decompress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_set_defaults referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_set_colorspace referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_set_quality referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_start_compress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg12_write_scanlines referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_finish_compress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_write_marker referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::CompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?CompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_read_header referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_start_decompress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg12_read_scanlines referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_finish_decompress referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_has_multiple_scans referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_set_marker_processor referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)
JPEG12_band.cpp.obj : error LNK2019: unresolved external symbol jpeg_resync_to_restart referenced in function "public: enum CPLErr __cdecl GDAL_MRF::JPEG_Codec::DecompressJPEG12(struct GDAL_MRF::buf_mgr &,struct GDAL_MRF::buf_mgr &)" (?DecompressJPEG12@JPEG_Codec@GDAL_MRF@@QEAA?AW4CPLErr@@AEAUbuf_mgr@2@0@Z)

JPEG12_band.cpp.obj : error LNK2001: unresolved external symbol "char * GDAL_MRF::CHUNK_NAME" (?CHUNK_NAME@GDAL_MRF@@3PADA)

JPEG12_band.cpp.obj : error LNK2001: unresolved external symbol "unsigned __int64 GDAL_MRF::CHUNK_NAME_SIZE" (?CHUNK_NAME_SIZE@GDAL_MRF@@3_KA)

gdald.dll : fatal error LNK1120: 21 unresolved externals

ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "e",
  "version": "1.0.0",
  "dependencies": [
    {
      "name": "crashrpt",
      "features": []
    }
  ]
}

```
</details>

--- a/ports/gdal/fix-jpeg.patch
+++ b/ports/gdal/fix-jpeg.patch
@@ -1,0 +1,13 @@
+diff --git a/cmake/helpers/CheckDependentLibraries.cmake b/cmake/helpers/CheckDependentLibraries.cmake
+index 9568ef043e..94617e3ac9 100644
+--- a/cmake/helpers/CheckDependentLibraries.cmake
++++ b/cmake/helpers/CheckDependentLibraries.cmake
+@@ -416,7 +416,7 @@ if (GDAL_USE_JPEG AND (JPEG_LIBRARY MATCHES ".*turbojpeg\.(so|lib)"))
+       "JPEG_LIBRARY should point to a library with libjpeg ABI, not TurboJPEG. See https://libjpeg-turbo.org/About/TurboJPEG for the difference"
+     )
+ endif ()
+-if (TARGET JPEG::JPEG)
++if (GDAL_USE_JPEG AND TARGET JPEG::JPEG)
+   set(EXPECTED_JPEG_LIB_VERSION "" CACHE STRING "Expected libjpeg version number")
+   mark_as_advanced(GDAL_CHECK_PACKAGE_${name}_NAMES)
+   if (EXPECTED_JPEG_LIB_VERSION)

--- a/ports/gdal/portfile.cmake
+++ b/ports/gdal/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_from_github(
         fix-gdal-target-interfaces.patch
         libkml.patch
         Fix-quote-variable.patch
+        fix-jpeg.patch
 )
 # `vcpkg clean` stumbles over one subdir
 file(REMOVE_RECURSE "${SOURCE_PATH}/autotest")

--- a/ports/gdal/vcpkg.json
+++ b/ports/gdal/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gdal",
   "version-semver": "3.7.1",
-  "port-version": 3,
+  "port-version": 4,
   "description": "The Geographic Data Abstraction Library for reading and writing geospatial raster and vector data",
   "homepage": "https://gdal.org",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2790,7 +2790,7 @@
     },
     "gdal": {
       "baseline": "3.7.1",
-      "port-version": 3
+      "port-version": 4
     },
     "gdcm": {
       "baseline": "3.0.22",

--- a/versions/g-/gdal.json
+++ b/versions/g-/gdal.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "eb6b9c542c7c01651906fee737d73fab64482700",
+      "version-semver": "3.7.1",
+      "port-version": 4
+    },
+    {
       "git-tree": "1bffcf8c8f2b02ed041a099723115cbbd0c39b2c",
       "version-semver": "3.7.1",
       "port-version": 3

